src/Chapter1.hs

```diff
@@ -619,8 +631,13 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = lastDig (abs n) + lastButOneDig (abs n)
+  where
+    lastDig :: Int -> Int
+    lastDig x = mod x 10
+    lastButOneDig :: Int -> Int
+    lastButOneDig x = mod (div x 10) 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that is actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(first, second) = divMod (mod x 100) 10
```

You can see how we could pattern match on the pair 🙂

Reply:
Hey, thanks a lot. I just wanted nudge to learn Haskell, this repo proved to be a really great exercise! Will push the other PR's soon after the weekend. :)